### PR TITLE
Support filter files by file name suffix

### PR DESCRIPTION
**Pull request by Linary**
Implement #44

Change-Id: I25484c390015daacd907ea102c31c1f8aaa8f495

### CODE REVIEW

File: src/main/java/com/baidu/hugegraph/loader/source/file/FileFilter.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    public Set<String> suffix() {
+        return this.suffix;
+    }
+
+    public boolean reserved(String name) {
+        if (this.suffix.size() == 1 && this.suffix.contains(ALL_SUFFIX)) {
+            return true;
+        }
+        return this.suffix.contains(FilenameUtils.getExtension(name));
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/loader/source/file/FileFilter.java]: this.suffix.endWith(FilenameUtils.getExtension(name))

Comment (Linary) [reply src/main/java/com/baidu/hugegraph/loader/source/file/FileFilter.java]: this.suffix is a set

File: src/main/java/com/baidu/hugegraph/loader/source/file/FileFilter.java [hunk 2]
```diff
@@ -31,20 +31,20 @@
     private static final String ALL_SUFFIX = "*";
 
     @JsonProperty("suffix")
-    private Set<String> suffix;
+    private Set<String> suffixes;
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/loader/source/file/FileFilter.java]: @JsonProperty("suffix") => @JsonProperty("extensions")

File: src/main/java/com/baidu/hugegraph/loader/source/file/FileSource.java [hunk 3]
```diff
@@ -35,6 +35,8 @@
 
     @JsonProperty("path")
     private String path;
+    @JsonProperty("filter")
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/loader/source/file/FileSource.java]: prefer file_filter

**Pull request merged**